about/index.html

```diff
@@ -57,7 +57,7 @@ <h1>About us</h1>
 <li><i class="fa-solid fa-paw"></i>&nbsp;&nbsp;Education: We have a ‘Congrats file’ that’s full of great information, including:</li>
  <li class="indent"> What is the best way to travel with my puppy?</li>
  <li class="indent"> What do I need to know about the drive/flight home?</li>
- <li class="indent"> We share tips to make the transition to home as smooth as possible while also creating good long-term habits</li>
+ <li class="indent"> Go home guidance for the transition to your home </li>
```

Comment:
This still sucks. Also if the format of the list is going to be questions, make them all questions consistently.

Get rid of the single empty space before the text in each of these elements. I also think somehow you overwrote the fix for the list somehow - the two indented lists on this page are still incorrect.